lib/routes/usuarios.js

```diff
@@ -26,5 +27,10 @@ router.get(
   [autenticar, permitirAcceso('admin', 'bedel', 'asistente')],
   withErrorHandling(getActividadesUsuario)
 );
+router.get(
+  '/yo/autorizaciones',
+  [autenticar, permitirAcceso('admin', 'bedel', 'asistente')],
```

Comment:
Solo tiene sentido para el `asistente`.

```suggestion
  [autenticar, permitirAcceso('asistente')],
```